lib/shopify_app/controller.rb

```diff
@@ -3,16 +3,6 @@
 module ShopifyApp
   module Controller
     extend ActiveSupport::Concern
-
     include ShopifyApp::LoginProtection
-
-    def fullpage_redirect_to(url)
-      if ShopifyApp.configuration.embedded_app?
-        render inline: %Q(<script type="text/javascript">window.top.location.href = #{url.to_json};</script>)
-      else
-        redirect_to url
-      end
-    end
-
```

Comment:
moving this makes sense to me :+1: 

At this point do we even need this ShopifyApp:Controller module? all it does is include ShopifyApp::LoginProtection